recipes/pypolyagamma-3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]
+then
+    export CC=gcc CXX=g++
+fi
+
+if [ $(uname) == Linux ]
+then
+    yum install gsl-devel
```

Comment:
Sorry to dup you @patricksnape. GitHub didn't show the comment in the diff when I had posted that.